doc/rbd/rbd-operations.rst

```diff
@@ -10,7 +10,7 @@
    Mirroring <rbd-mirroring>
    Live-Migration <rbd-live-migration>
    Persistent Read-only Cache <rbd-persistent-read-only-cache>
-   Persistent Write-back Cache <rbd-persistent-write-back-cache>
+   Persistent Write Log <rbd-persistent-write-log>
```

Comment:
@CongMinYin minor nit: Persistent Write Log is still a form of cache for RBD, to someone not familiar with PWL, I think it would be good to have Cache specified as "Persistent Write Log Cache"

Reply:
Make sense, done. `Write back cache` is the earliest name. It was later named `write log` according to the design and implementation. Now, according to our discussion with you, we tend to call it `rbd persistent cache`. The name of `rbd persistent write log cache` is more detailed.